tests/test_matrix_utils.py

```
@@ -3035,26 +3011,46 @@ def e2ee_config(self):
             # Mock AsyncClient instance with proper async methods
             mock_client_instance = MagicMock()
             mock_client_instance.rooms = {}
-            mock_client_instance.login = AsyncMock(return_value=MagicMock())
-            mock_client_instance.sync = AsyncMock(return_value=MagicMock())
-            mock_client_instance.join = AsyncMock(return_value=MagicMock())
-            mock_client_instance.close = AsyncMock()
-            mock_client_instance.get_displayname = AsyncMock(
-                return_value=MagicMock(displayname="TestBot")
-            )
+
+            async def mock_login(*args, **kwargs):
```

Ruff (0.14.3)

Unused function argument: `args`

(ARG001)